mlflow/R/mlflow/man/mlflow_run.Rd

```diff
@@ -6,8 +6,7 @@
 \usage{
 mlflow_run(entry_point = NULL, uri = ".", version = NULL,
   param_list = NULL, experiment_id = NULL, experiment_name = NULL,
-  mode = NULL, cluster_spec = NULL, git_username = NULL,
-  git_password = NULL, no_conda = FALSE, storage_dir = NULL)
+  backend = NULL, backend_spec = NULL, no_conda = FALSE, storage_dir = NULL)
```

Comment:
Did you make this change manually? haha, we should figure out how to regenerate the Rdoc soon... @smurching

Reply:
yep xD